labs/lab08-airgap-lite/README.md

```diff
@@ -302,73 +304,72 @@ You must deploy a new release in order to fix this error.
 ***
 ## Deploying a new version
 
-As part of the lab setup, a new release was created in the Vendor Portal with the fix. To make the release available:
+As part of the lab setup, a new release was created in the vendor portal with the fix. To make the release available:
 1. Navigate to **Sequence 2** under **All releases**.  
 1. Click **Promote**.
 1. Select the `lab08-airgap-lite` channel as the channel to promote the release to.
 
    ![app-down](img/promote-sequence-2.png)
 
-If you are interested, you can review the differences between the two releases in the Vendor Portal. It is also shown below:
-
-```diff
-diff --git a/labs/lab05-airgap/manifests/nginx-deployment.yaml b/labs/lab05-airgap/manifests/nginx-deployment.yaml
-index fa29e8d..3a66405 100644
---- a/labs/lab05-airgap/manifests/nginx-deployment.yaml
-+++ b/labs/lab05-airgap/manifests/nginx-deployment.yaml
-@@ -16,9 +16,6 @@ spec:
-       containers:
-         - name: nginx
-           image: nginx:latest
--          command:
--            - exit
--            - "1"
-           volumeMounts:
-             - mountPath: /usr/share/nginx/html
-               name: html
+1. (Optional) If you are interested, you can review the differences between the two releases in the vendor portal. It is also shown below:
+
+   ```diff
+   diff --git a/labs/lab05-airgap/manifests/nginx-deployment.yaml b/labs/lab05-airgap/manifests/nginx-deployment.yaml
+   index fa29e8d..3a66405 100644
+   --- a/labs/lab05-airgap/manifests/nginx-deployment.yaml
+   +++ b/labs/lab05-airgap/manifests/nginx-deployment.yaml
+   @@ -16,9 +16,6 @@ spec:
+          containers:
+            - name: nginx
+              image: nginx:latest
+   -          command:
+   -            - exit
+   -            - "1"
+              volumeMounts:
+                - mountPath: /usr/share/nginx/html
+                  name: html
 ```
 
-When the promotion is done, you should be able to navigate back to the customer download portal we accessed from the customer page.
-Scrolling to the bottom, you can click **Show older bundles** to see the history of the releases on the `lab08-airgap` channel.
-
+1. When the promotion is done, navigate back to the customer download portal we accessed from the customer page.
+Scroll to the bottom and click **Show older bundles** to see the history of the releases on the `lab08-airgap` channel.
 
-The new release may take a minute or two to build. You can refresh the page until you see a release with a timestamp that matches when you promoted your release.
+   The new release can take a minute or two to build. Refresh the page until you see a release with a timestamp that matches when you promoted your release.
 
-![download-portal-more](img/download-portal-more.png)
+   ![download-portal-more](img/download-portal-more.png)
```

Comment:
Missing indent one line below for `When you've downloaded the new version, you can now deploy it:`